pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -442,6 +449,16 @@ func (c *InstallerController) manageInstallationPods(ctx context.Context, operat
 		return true, requeueAfter, nil
 	}
 
+	// Check if revision should be installed based on if the quorum is about to be violated
+	if c.shouldRevisionInstall != nil {
+		quorumSafe, err := c.shouldRevisionInstall()
```

Comment:
I would avoid leaking our usage of "quorum" in the variable name, just call it `shouldInstall`